### PR TITLE
Added Spanish translations

### PR DESCRIPTION
**Pull request by Parodper**
Notes:

**edlin**

FreeDOS does have a translated HELP file, but it is not present on its github DOC folder.

**fdi**

Is there something to translate here?

**fdi-x86**

There might be some problems about the action of the translated verbs.
Also, seems weird how «FreeDOS» is a variable only on most cases. Is this from when it was still called PDOS?

### CODE REVIEW

File: fdi-x86/nls/SETUP.ES [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+WELCOME_0=/p
+WELCOME_1=/fLightGreen "%1" /fGray es un sistema operativo completo. Si escoje instalarlo en su ordenador puede ser que sobrescriba el sistema operativo que ya tenga (por /p ejemplo, Windows.)
+WELCOME_2=/c32 Si no quiere que pase eso salga ahora.
+WELCOME_3=
+WELCOME_4=
+WELCOME_5=/p
+
+HRULE=/fDarkGray /x 0xC4 /fGray
+
+CONTINUE=/p �Quiere continuar? /c32
```
Comment (shidel) [inline fdi-x86/nls/SETUP.ES:13]: Issue--Open ended quote

File: fdi-x86/nls/SETUP.ES [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+REBOOT=/p �Quiere reiniciar ahora? /c32
```
Comment (shidel) [inline fdi-x86/nls/SETUP.ES:14]: Issue--open ended quote

**Comment by shidel** [comment]
> **edlin**
> FreeDOS does have a translated HELP file, but it is not present on its github DOC folder.

I'm unsure what you are referring too. Could you clarify and possibly provide links?

> **fdi**
> Is there something to translate here?

Not really, it was originally translated with my limited broken Spanish and Google Translate. The later, a volunteer fixed a bunch of grammatical errors. But, there could be more issues with the translation. So, it wouldn't hurt going over it. [FDI SPANISH TRANSLATION](https://github.com/shidel/fd-nls/blob/master/fdi/language/es/FDSETUP.DEF)

> **fdi-x86**
> There might be some problems about the action of the translated verbs.
> Also, seems weird how «FreeDOS» is a variable only on most cases. Is this from when it was still called PDOS?

Opps, I thought there were some open ended quotations. But, I was wrong. It was how the web interfaces to GitHub was rendering the upside-down question mark. I apologize for any confusion. Skimming them, they look good. :-)

The translations for batch files are a little painful to translate. But, unfortunately it is unavoidable without a severe impact on performance. If you have a concern about one of them, you can fairly easily test them in FreeDOS using vecho. 

`vecho /T filename id %1 %2 %3...`

filename is the file, ID is the string (case-specific) then followed by any variables you need. For example, to test the do_install resource, you would enter something like this:

`vecho /t setup.es DO_INSTALL one two`

If something is wrong with the resource, you'll get a fairly unhelpful `parameter error` message. But, at least it lets
you know when it encounters any formatting errors and it may output some of the text to help find the issue.

On a  side note...

There is a special directory called [packages](https://github.com/shidel/fd-nls/tree/master/packages). It contains CSV files for different languages that are used by programs like FDIMPLES and PKGINFO to provide multi-language descriptions for the packages. Eventually, the [Official FreeDOS Software Repository](https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/repositories/latest/pkg-html/) management utilities will support and generate custom user specific language versions of the HTML. At present, there are only translations for DE, TR and FR.

 If your interested in translating those, just grab the [master.csv](https://github.com/shidel/fd-nls/blob/master/packages/master.csv) and save it to a new sub-directory as a listing.csv file. The dir structure isn't difficult to figure out. Just take a quick look at the other language translations in there. Once there is a ES version, in the future, the utility will create a mixed.csv file. That file will contain all the previous translations for that language. Also, it will have the english version for any new packages denoted with a star in a "new" column.

**Comment by Parodper** [comment]
> > **edlin**
> > FreeDOS does have a translated HELP file, but it is not present on its github DOC folder.
> 
> I'm unsure what you are referring too. Could you clarify and possibly provide links?

When I wanted to try to get HELP to display my translated file I found out it was already translated, with 2020 copyright. Now that I am looking more into it, seems like HELP has its own version of the HTM files of some programs inside the HHSTNDRD.ZIP file. In the [HTMLHELP](https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/repositories/1.3/base/htmlhelp/) package it is inside the HELP/ES/HHSTNDRD folder.

**Comment by shidel** [comment]
Ok. However, I am unsure what your asking me to do. Are you asking me to copy the HTMLHELP html page to the NLS for EDLIN? Or, the other way around? Or, just an FYI?

**Comment by Parodper** [comment]
It was half of a FYI, and half asking what should I do (copy the translation or leave that as-is). Now I'm wondering if hhstndrd should be compressed, or if it can be compressed when the HTMLHELP package is built.

**Pull request merged**